pkg/envoy/envoy_test.go

```diff
@@ -72,15 +74,19 @@ func (s *EnvoySuite) TestEnvoy(c *C) {
 		c.Skip("skipping envoy unit test; CILIUM_ENABLE_ENVOY_UNIT_TEST not set")
 	}
 
-	stateLogDir := c.MkDir()
+	flowdebug.Enable()
```

Comment:
Remove this line before merging.

Reply:
This test is normally skipped, you have to have `CILIUM_ENABLE_ENVOY_UNIT_TEST` defined to run it. Given this I'd leave the flowdebug in there, presuming that this test would typically be run alone.